katas/main/src/pokerhands/jorge/4.Score.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace PokerGame.Score.Jorge;
+using PokerGame.Dictionary.Jorge;
+using PokerGame.Enums.Jorge;
+
+public class PokerGameScoreGetter
+{
+    public string[] resultValues = new string[4]; // Array for results
+
+    public string[] Score(
```

Comment:
This method is doing too much, it would be a good idea to break it in smaller pieces